### DISCUSSION TITLE
some plasma-content just got merged into home-manager

### DISCUSSION THREAD

**Discussion by Asqiir** [nix-community/plasma-manager#134] [General]
There is some new plasma-related content in home-manager: https://github.com/nix-community/home-manager/pull/2565

Is it interesting for us? Do we have duplicate code now? Do we have obsolete code now?

**Comment by magnouvean** [comment]
This might provide an interface which we can use in `plasma-manager`. It will need some testing. Thanks for sharing, it might become useful.

**Reply by magnouvean** [reply]
Although it does look like it uses kwriteconfig and is thus quite limited by that. We could take some inspiration by the qdbus calls though and see if this could help this project in some way.
